src/coreclr/dlls/mscordbi/CMakeLists.txt

```diff
@@ -106,6 +106,12 @@ elseif(CLR_CMAKE_HOST_UNIX)
         )
     endif(CLR_CMAKE_HOST_HAIKU)
 
+    if(CLR_CMAKE_TARGET_ANDROID)
+        list(APPEND COREDBI_LIBRARIES
+            log
+        )
+    endif(CLR_CMAKE_TARGET_ANDROID)
+
```

Comment:
@grendello @janvorli @lateralusX 
I suggest we move the logging bits into a separate PR which would contribute to: https://github.com/dotnet/runtime/issues/112865 I just created